source/interfaces/artifact-api.rst

```diff
@@ -30,22 +51,11 @@ While we recommend working with QIIME 2 ``Artifacts`` directly, it is possible t
    51121722488d0c3da1388d1b117cd239	0.0	0.0	0.0	0.0	0.0
    1016319c25196d73bdb3096d86a9df2f	11.0	17.0	12.0	4.0	2.0
 
-You can also view the artifact's data as a ``pandas.DataFrame`` object:
-
-.. code-block:: python
 
-   >>> import pandas as pd
-   >>> df = rarefied_table.view(pd.DataFrame)
-   >>> df.head()
-           b32621bcd86cb99e846d8f6fee7c9ab8  99647b51f775c8ddde8ed36a7d60dbcd  \
-   L1S105                              25.0                               0.0
-   L1S140                              31.0                               0.0
-   L1S208                              27.0                               0.0
-   L1S257                              29.0                               0.0
-   L1S281                              23.0                               0.0
-   ...
+A powerful feature of QIIME 2 is that you can export different types of views from QIIME artifacts as illustrated here, then operate on the resulting data types, and import those data back into QIIME. This is useful if there are some operations that are available on the view's data type (e.g., the ``pandas.DataFrame``) that are not available through the QIIME API.
```

Comment:
```suggestion
A powerful feature of QIIME 2 is that you can export different types of views from QIIME 2 artifacts as illustrated here, then operate on the resulting data types, and import those data back into QIIME 2. This is useful if there are some operations that are available on the view's data type (e.g., the ``pandas.DataFrame``) that are not available through the QIIME 2 API.
```
Looks like we have some leftover descriptions from QIIME 1 in this paragraph - since you're updating this segment of the docs anyways, this is a good tweak to add in there so that our language is consistent in listing QIIME 2 instead of QIIME.